Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -671,9 +671,10 @@ void WebProcessProxy::setThirdPartyCookieBlockingMode(ThirdPartyCookieBlockingMo
 
 Ref<WebPageProxy> WebProcessProxy::createWebPage(PageClient& pageClient, Ref<API::PageConfiguration>&& pageConfiguration)
 {
-    Ref<WebPageProxy> webPage = WebPageProxy::create(pageClient, *this, WTFMove(pageConfiguration));
+    auto isBlobPartitioningEnabled = isBlobRegistryPartitioningEnabled(*pageConfiguration->preferences());
+    Ref<WebPageProxy> webPage = WebPageProxy::create(pageClient, *this, WTFMove(pageConfiguration), isBlobPartitioningEnabled);
```

Comment:
We shouldn't be passing `isBlobPartitioningEnabled` to `WebPageProxy::create()` I think. We already paid the pageConfiguration and isBlobPartitioningEnabled comes from the pageConfiguration.